client/src/modules/Mentor/components/TaskStatusTabs/TaskStatusTabs.test.tsx

```diff
@@ -1,8 +1,8 @@
 import { fireEvent, render, screen } from '@testing-library/react';
 import TaskStatusTabs, { Status } from './TaskStatusTabs';
-import { StudentTaskSolutionItemStatus, TASKS_STATUSES } from 'modules/Mentor/constants';
+import { SolutionItemStatus } from 'modules/Mentor/constants';
 
-const StatusEnum = StudentTaskSolutionItemStatus;
+const StatusEnum = SolutionItemStatus;
```

Comment:
Let's remove this, it's really unnecessary, especially taking into account that you use imported value name on line 34 :)